index.js

```diff
@@ -56,7 +56,13 @@ module.exports = async (target, options) => {
 	} else if (process.platform === 'win32' || isWsl) {
 		command = 'cmd' + (isWsl ? '.exe' : '');
 		cliArguments.push('/c', 'start', '""', '/b');
-		target = target.replace(/&/g, '^&');
+
+		// Always quoting target allows for urls/paths to have spaces and unmarked characters, as cmd will
+		// interpret them as plain text to be forwarded as one unique argument. Enabling windowsVerbatimArguments
+		// disables node's default quotes and escapes handling (https://git.io/fjdem).
+		// References: Issues #17, #44, #55, #77, #101 and #115 / Pull requests: #74 and #98
+		target = `"${target}"`;
```

Comment:
Don't we need to escape any double-quotes in `target`? Can you also add a test for a target containing a double-quote?

Reply:
Quick update: I should pick this up again tonight, things have been hectic here too in the past few days.

Reply:
Hey @sindresorhus, I could not make any number of double-quotes go through. Command prompt can only do 1 of 2 things: handle them all **OR** not handle them at all. As per its own documentation:

>*If /C or /K is specified, then the remainder of the command line after the switch is processed as a command line, where the following logic is used to process quote (") characters:*
>
>    1.  *If all of the following conditions are met, then quote characters on the command line are preserved:*
>        - *no /S switch*
>        - *exactly two quote characters*
>        - *no special characters between the two quote characters, where special is one of: &<>()@^|*
>        - *there are one or more whitespace characters between the two quote characters*
>        - *the string between the two quote characters is the name of an executable file.*
>
>    2.  *Otherwise, old behavior is to see if the first character is a quote character and if so, strip the leading character and remove the last quote character on the command line, preserving any text after the last quote character.*

So, since we were already shooting for the second option, I've also added the `/s` switch so we enforce consistency. The final behavior is to handle double-quotes correctly, but that will also mean that all double-quotes in your command/url will magically disappear (in the url cases you could still use `encodeURL` to encode the double-quotes, which will make them get to the destination just fine).

Reply:
We can leave it like this for now and if someone complain we can possibly add an option. Can you at least document the behavior so it's clear to people that they need to use `encodeURL` for URLs and that double-quotes be "stripped". It's unlikely that anyone would ever have it in their file paths.

Reply:
Actually, how about a `{url: true}` or `{type: 'url'}` option which handles the encoding for the user so they don't have to encode the URL themselves?

Reply:
I've done both :)